matfact/pyproject.toml

```diff
@@ -43,6 +44,7 @@ plugins = "numpy.typing.mypy_plugin"
 [[tool.mypy.overrides]]
 module = [
     "matfact.*",
+    "processing.*",
```

Comment:
Why do we ignore `matfact` and `processing` with mypy?

Reply:
Because of mypy:
`matfact/preprocessing/data_manager.py:5: error: Skipping analyzing "processing.pipelines": module is installed, but missing library stubs or py.typed marker`

Removed the ignore of itself though.